spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -52,6 +52,42 @@
     end
   end
 
+  context "a ActiveModel that's validate a delegated attribute to a ActiveModel" do
```

Comment:
What about:

"an ActiveModel with a validation on an attribute that delegates to an association in another ActiveModel"